release_autogen_client/common_packaging_utils.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Usage: $ source ./common_packaging_utils.sh
+
+set -e
+set -x
+
+export MARQUEZ_CLONE_DIR="/tmp/marquez"
+export MARQUEZ_PYTHON_CODEGEN_CLONE_DIR="/tmp/marquez-python-codegen"
+export OPEN_API_GENERATOR_CLONE_DIR="/tmp/openapi_generator"
+export CONFIG_FILE_LOCATION="${MARQUEZ_PYTHON_CODEGEN_CLONE_DIR}/config.json"
+
+clone_marquez_python_client_codegen()
+{
+  clone_dir=${MARQUEZ_PYTHON_CODEGEN_CLONE_DIR}
+  rm -rf ${clone_dir} || true
+  git clone https://github.com/MarquezProject/marquez-python-codegen.git ${clone_dir}
+}
+
+clone_marquez()
+{
+  clone_dir=${MARQUEZ_CLONE_DIR}
+  rm -rf ${clone_dir} || true
+
+  git clone --depth=1 https://github.com/MarquezProject/marquez.git ${clone_dir}
+}
+
+update_config_file()
+{
+  sed -i '' "s/packageVersion.*/packageVersion\": \"${version}\",/g" ${CONFIG_FILE_LOCATION}
```

Comment:
@wslulciuc Updated to use the existing config file. Verified that the produced diff in the `marquez-python-codegen` project contains `config.json` as well, so that there will be a record of what was used to auto-generate the rest of the client.